### DISCUSSION TITLE
Large residual errors with AIMD data

### DISCUSSION THREAD

**Discussion by RicardoCecconello** [ttadano/alamode#172] [Q&A] [answered]
Dear Dr. Tadano,

I'm doing some calculations of the IFCs with 290 snapshots from AIMD with `LMODEL = enet` and `LMODEL = adaptive-lasso` and the residual errors are huge (above 50 %). One issue that I observed is that almost every time I get the error _Please use a smaller CV_MINALPHA_. I've reduced it to 1.0e-16 and used `CV_NALPHA = 100`, but I still get the error in some of the CV sets. This occurs mostly when `LMODEL = enet`, where I have also verified that changing the `L1_RATIO` value doesn't solve the problem. It happens for different CV values as well. Any ideas on the cause of this problem?

I also checked how the displacement of the atoms (compared to the equilibrium structure) changes on the AIMD trajectory, and I observed that one of the components has a mean displacement 4 times greater than the other ones. Could this be a cause of the problem I'm facing?

Thanks in advance,
Ricardo.

**Comment by ttadano** [comment, marked as answer]
The large fitting error you observed might be result from the large displacement you mentioned and/or low flexibility of the model. 

Can you check if the fitting error is still large with `LMODEL = ols`? 

If the error is still as large as 50% and you need to reduce it further, you will have to make the model more flexible by increasing `NORDER` or using larger cutoff radii.

**Reply by RicardoCecconello** [reply]
With `LMODEL = ols` the error is similar. We'll try with higher `NORDER` and cutoff. Thank you!
